nixos/modules/services/monitoring/riemann.nix

```diff
@@ -84,6 +100,8 @@ in {
         User = "riemann";
         ExecStart = "${launcher}/bin/riemann";
       };
+      serviceConfig.LimitNOFILE = 65536;
+      restartTriggers = [ configFile ];
```

Comment:
Since the launcher will change every time the config file changes, there's no need to add it to the restart triggers manually.